docs/remote_inference_blueprints/cohere_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,361 @@
+### Cohere Embedding Connector Blueprint:
+
+This blueprint will show you how to connect a Cohere embedding model to your Opensearch instance, including creating a k-nn index and your own Embedding pipeline. You will require a Cohere API key.
+
+Cohere currently offers the following Embedding models (with model name and embedding dimensions). Note that only the following have been tested with the blueprint guide.
+
+- embed-english-v3.0 1024
+- embed-english-v2.0 4096
+
+See [Cohere's /embed API docs](https://docs.cohere.com/reference/embed) for more details.
+
+#### 1; Update your Opensearch cluster settings
+
+```json
+PUT /_cluster/settings
+
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+          "^https://api\\.cohere\\ai/.*$",
```

Comment:
We already added this Cohere URl pattern to default trsuted endpoints list,  don't need to ask user do this again.

Reply:
Sg